prepare.sh

```diff
@@ -44,24 +44,37 @@ echo ""
 echo "replacing slug:"
 FILES=$(find . -type f ! -name 'prepare.sh' ! -wholename '*node_modules*' -print0 | xargs -0 grep -l "$REPLACE_SLUG")
 echo ${FILES} | tr ' ' '\n'
-echo ${FILES} | xargs sed -i "" -e "s/$REPLACE_SLUG/$SERVICE_SLUG/g"
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    echo ${FILES} | xargs sed -i "" -e "s/$REPLACE_SLUG/$SERVICE_SLUG/g"
+else
+    echo ${FILES} | xargs sed -i -e "s/$REPLACE_SLUG/$SERVICE_SLUG/g"
+fi
```

Comment:
on both linux and MacOs should work:
`echo ${FILES} | xargs sed -i"" -e "s/$REPLACE_SLUG/$SERVICE_SLUG/g"`

<- see no space after `-i`

can you try it?